kube/helm/carbonj/templates/persistent-volume-claims.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.carbonj.namespace }}-p{{ .Values.carbonj.shard.id }}-data-carbonj-p{{ .Values.carbonj.shard.id }}-0
+  namespace: {{ .Values.carbonj.namespace }}
+  labels:
+    app: carbonj
+  annotations:
+    volume.beta.kubernetes.io/storage-class: {{ .Values.carbonj.storageClass }}-{{ .Values.carbonj.region }}{{ .Values.carbonj.availabilityZone }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ .Values.carbonj.diskSize }}
+#---
```

Comment:
So this will only support one replica?

Reply:
I could uncomment to produce two replicas or leave a block comment saying that uncommenting would produce 2 replicas and they'd have to update carbonj.nodes in the values to match the number of replicas as well.

Reply:
This actually brought up something else - if they are running two replicas per shard we should provide a way to create those in different AZs.